.github/workflows/knative-boilerplate.yaml

```diff
@@ -30,15 +30,15 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         extension:
-        - go
-        - sh
+          - go
```

Comment:
Was it formatted with IDE maybe? It will be overriden by GH action sync anyway, just curious.

Reply:
Yes, the IDE formatted it.